libs/platform/ebpf_epoch.c

```diff
@@ -323,95 +332,73 @@ ebpf_epoch_terminate()
     EBPF_RETURN_VOID();
 }
 
-ebpf_epoch_state_t*
-ebpf_epoch_enter()
+#pragma warning(push)
+#pragma warning(disable : 28167) // Not sure how to annotate that _ebpf_epoch_raise_to_dispatch_if_needed may raise
```

Comment:
since the function also unconditionally restores the irql what is the warning about?